meinberlin/apps/projects/templates/meinberlin_projects/includes/timeline.html

```diff
@@ -14,7 +15,7 @@
                         href="{{ project.get_absolute_url }}"
                         class="button button--light button--small timeline__button {% if obj == active_phase.module %}is-active{% endif %} {% if not object|is_offlineevent %}is-selected{% endif %}"
```

Comment:
There are several places in the code that are not yet ready for multi-modules. This is also because there are not yet sufficient interfaces. I have a general outline of the necessary steps in my head.